src/config/api.rs

```diff
@@ -151,7 +158,7 @@ pub(super) fn api_keys_path() -> PathBuf {
 
 pub(super) fn generate_api_keys_file() -> std::io::Result<()> {
     let mut api_config = HashMap::new();
-    api_config.insert(Api::Ollama.to_string(), ApiConfig::openai());
+    api_config.insert(Api::Ollama.to_string(), ApiConfig::ollama());
```

Comment:
Randomly stumbled upon this as well, I know it doesn't affect behavior right now since the default config overwites it anyway but I figured I may as well point it to the correct API just in case